client/my-sites/stats/all-time-views-section/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo, useState } from 'react';
+import { useSelector } from 'react-redux';
+import QuerySiteStats from 'calypso/components/data/query-site-stats';
+import SimplifiedSegmentedControl from 'calypso/components/segmented-control/simplified';
+import { getSiteStatsViewSummary } from 'calypso/state/stats/lists/selectors';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import Months from '../stats-views/months';
+
+import './style.scss';
+
+type chartOption = {
+	value: string;
+	label: string;
+	path: string;
+};
+
+export default function AllTimeViewsSection( { siteId, slug }: { siteId: number; slug: string } ) {
+	const query = { quantity: -1, stat_fields: 'views' };
+	const translate = useTranslate();
+	const [ chartOption, setChartOption ] = useState( 'total' );
+	const viewData = useSelector( ( state ) => getSiteStatsViewSummary( state, siteId ) );
+	const monthViewOptions = useMemo( () => {
+		return [
+			{ value: 'total', label: translate( 'Months and years' ) },
+			{ value: 'average', label: translate( 'Average per day' ) },
+		];
+	}, [ translate ] );
+
+	const toggleViews = ( option?: chartOption ) => {
+		setChartOption( option?.value || '' );
```

Comment:
What happens when `chartOption` is set to `''`?

Reply:
Yes, this default value should be `total`, the same as the initial state.